.github/workflows/conformance.yml

```diff
@@ -10,6 +10,13 @@ on:
       antrea-values:
         description: The Antrea Chart values. Multiple values can be separated with commas (e.g. key1=val1,key2=val2). Default configuration will be tested if empty.
         required: false
+      distro:
+        description: The image distro to test. It could be ubuntu or ubi.
+        type: choice
+        options:
+          - ubuntu
+          - ubi
+        default: ubuntu
```

Comment:
should we make this `antrea-image-repository` for more flexibility?

Reply:
changed to repository. Thanks for the suggestion.